cleaner/scheduled_tasks/classes/form/task_form.php

```diff
@@ -0,0 +1,107 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * @package     cleaner_scheduled_tasks
+ * @copyright   2019 Catalyst IT Australia {@link http://www.catalyst-au.net}
+ * @license     http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+namespace cleaner_scheduled_tasks\form;
+use html_writer;
+use moodleform;
+
+defined('MOODLE_INTERNAL') || die();
+
+global $CFG;
+require_once("{$CFG->libdir}/formslib.php");
+
+class task_form extends moodleform {
+    protected function definition() {
+        $mform = $this->_form;
+        $tasks = $this->_customdata;
+
+        if (!$tasks) {
+            throw new \coding_exception(get_string('noscheduledtasks', 'cleaner_scheduled_tasks'));
+        }
+        get_string('pluginname', 'cleaner_scheduled_tasks');
+        // Display a header on the page.
+        // Need to put lang strings into lang folder later.
+        $header = html_writer::tag('h2', get_string('title', 'cleaner_scheduled_tasks'), ['class' => 'scheduled_task_header']);
+        $headersubtitle = html_writer::tag('p', get_string('subtitle', 'cleaner_scheduled_tasks'), ['class' => 'scheduled_task_header_subtitle']);
+
+        $headerarray = [];
+        $headerarray[] = &$mform->createElement('static', 'stitle', 'stitle', "$header $headersubtitle");
+        $mform->addGroup($headerarray, 'header_array', '' , ' ', false);
+
+        // Header for each column
+        $mform->addElement('static', 'description', 'Component', 'Task Name');
+
+        global $DB;
+        $cleanertasks = $DB->get_records_sql("select * from {cleaner_scheduled_tasks} cs
+												join {task_scheduled} ts on ts.id = cs.task_scheduled_id");
```

Comment:
whitespace ?

Also try to keep all the sql keywords in caps

https://docs.moodle.org/dev/SQL_coding_style#Parameter_placeholders